src/ErrorBoundaryProvider.tsx

```diff
@@ -1,15 +1,12 @@
-import { ICoreReportingAPI } from '@stoplight/reporter';
 import * as React from 'react';
 import { FallbackComponent } from './FallbackComponent';
 import { FallbackProps } from './types';
 
 export type ErrorBoundaryContext = {
-  reporter: ICoreReportingAPI;
   FallbackComponent?: React.ElementType<FallbackProps>;
 };
 
 export const ErrorBoundaryContext = React.createContext<ErrorBoundaryContext>({
```

Comment:
One more note - I believe we don't quite need it. While having a custom default fallback might be nice to have, I believe the existing fallback we use is good enough, and we don't really need to override it globally.

Reply:
I got rid of it. Let's have simpler code.

Reply:
Ah, we actually have some custom usage of it in here packages/ninja/src/components/Fallback.tsx
https://github.com/stoplightio/platform-internal/blob/v2/packages/ninja/src/components/Fallback.tsx.
Alright, gonna keep it. Can delete later on if needed. It's actually prob nicer to have "contact Stoplight team" in the context of Platform, but less so overall (public usage of Elements, etc)